### ISSUE TITLE
Spotify can't be installed anymore

### ISSUE THREAD

**Issue by Vistaus** [solus-project/3rd-party#515]
Seems like the source has been moved:

```
13:28:23 Downloads $ sudo eopkg bi --ignore-safety https://raw.githubusercontent.com/solus-project/3rd-party/master/multimedia/music/spotify/pspec.xml
                     
Password: 
pspec.xml                      (13.0 KB)100%     32.10 MB/s [00:00:00] [complete]
actions.py                     (490.0  B)100%      7.54 MB/s [00:00:00] [complete]
DEBUG: InstallDB initialized in 0.00598192214966.
DEBUG: RepoDB initialized in 6.69956207275e-05.
DEBUG: Remote file can not be reached. Previously downloaded part of the file will be removed.
Building source package: spotify
component.xml                  (48.0  B)100%    677.96 KB/s [00:00:00] [complete]
Fetching source from: https://repository-origin.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.0.89.313.g34a58dea-5_amd64.deb
DEBUG: Remote file can not be reached. Previously downloaded part of the file will be removed.
Trying fallback address: https://www.solus-project.com/source/spotify-client_1.0.89.313.g34a58dea-5_amd64.deb
```

And while it tries a fallback address, it keeps hanging for like 30 minutes without actually downloading anything from the fallback address.

Please resolve this issue :)

**Comment by JoshStrobl** [comment]
We switched over to https://github.com/getsolus/solus-sc and https://github.com/getsolus/3rd-party

With that, the newest SC in unstable will be using the new 3rd-party repo. I'll go ahead and backport the Spotify update though to sort it out until tomorrow's sync to stable repo.

**Comment by Vistaus** [comment]
But I'm already using the Unstable repo and received the new SC this morning...

**Comment by JoshStrobl** [comment]
Should be resolved as of 6befce2bb19685dda8f21fae5f7fcd67ac94755f

> But Spotify also doesn't install the graphical way because of the same error, so apparently, the graphical way is also broken

Yea, this is a known issue.

> despite the fact that I'm using the new Solus SC from the Unstable repo.

That doesn't magically fix the fact if it fails to fetch it then it'll lock up.

**Comment by Vistaus** [comment]
Eh, okay. But thanks for fixing the issue, it works now :)
